src/js/rtc_session.js

```diff
@@ -437,7 +437,9 @@ export default class RtcSession {
             throw new IllegalParameters('iceServers required');
         }
         if (typeof contactToken !== 'string' || contactToken.trim().length === 0) {
```

Comment:
Maybe just remove this block. Take the input as it is.

Reply:
Removed and updated pull request